### PR TITLE
feat(Icons): Add GraphStatsAscendIcon

### PR DESCRIPTION
**Pull request by jcenglish**
### Overview

Adds GraphStatsAscendIcon and updates Icon docs with link to Google Drive assets folder.

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/E7CZLBi8CNgovH5mN1ivZz/Concept-Refinement?node-id=208%3A5333
- [x] Related to JIRA ticket: EGG-1019
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

File: packages/styleguide/stories/Atoms/Icons/Regular.stories.mdx [hunk 1]
```diff
@@ -47,7 +47,7 @@ If you cannot find the Icon you would like in the ones that we have specified he
 
 1. With design's approval pick an Icon ideally from `Streamline Regular`, if you need a filled icon you may pick from `Streamline Bold` but we would prefer the former.
    If you need something at exists in neither please consult design to create a custom one.
-2. Select the Outline version of the icon as opposed to stroke. You will not be able to get this from the App, and you must find the icon's outline version in our assets.
+2. Select the Outline version of the icon as opposed to stroke. You will not be able to get this from the App, and you must find the icon's outline version in [our assets](https://drive.google.com/drive/folders/1y0etPCUHGeU19Jp2UK0IG7Ku0oGka095?usp=sharing).
```
Comment (jcenglish) [inline packages/styleguide/stories/Atoms/Icons/Regular.stories.mdx:50]: Is there a reason not to include this link?

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.6.1-alpha.a93c96.0<br/>@codecademy/gamut-kit@0.3.19-alpha.a93c96.0<br/>@codecademy/gamut-labs@13.4.11-alpha.a93c96.0<br/>@codecademy/gamut@29.9.9-alpha.a93c96.0<br/>@codecademy/styleguide@33.9.11-alpha.a93c96.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6077383699c20f1f42fcefc2--gamut-preview.netlify.app>https://6077383699c20f1f42fcefc2--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6077383699c20f1f42fcefc2>Deploy Logs</a></p>

**Review by saghdaey — APPROVED** [review]

File: packages/styleguide/stories/Atoms/Icons/Regular.stories.mdx (hunk 1 above)
Comment (saghdaey) [reply packages/styleguide/stories/Atoms/Icons/Regular.stories.mdx:50]: i defer to webplat on this:)

Comment (JoshuaKGoldberg) [reply packages/styleguide/stories/Atoms/Icons/Regular.stories.mdx:50]: @codecaaron 😄

Comment (codecaaron) [reply packages/styleguide/stories/Atoms/Icons/Regular.stories.mdx:50]: Bleh we need a place to distribute the zip file its a 10000x easier to use than the drive.

Comment (jcenglish) [reply packages/styleguide/stories/Atoms/Icons/Regular.stories.mdx:50]: I'll remove this then.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.6.1-alpha.ba6206.0<br/>@codecademy/gamut-kit@0.3.25-alpha.ba6206.0<br/>@codecademy/gamut-labs@13.5.2-alpha.ba6206.0<br/>@codecademy/gamut@30.1.2-alpha.ba6206.0<br/>@codecademy/styleguide@34.2.2-alpha.ba6206.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6078553046afef55e4fcef31--gamut-preview.netlify.app>https://6078553046afef55e4fcef31--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6078553046afef55e4fcef31>Deploy Logs</a></p>

**Pull request merged**